src/renderers/render-doc.js

```diff
@@ -14,7 +14,8 @@ export default async function renderDoc(path) {
     const md = response.data[0].FullBody;
     const meta = response.data[0].FullMeta;
```

Comment:
I would prefer creating a new js file alongside index.js - doc-page-types.js
```
const docPageType = {
  DOC_ARTICLE: 'doc-article',
  DOC_LANDING: 'doc-landing',
  SOLUTION_LANDING: 'solution-landing'
};
```
In the renderDoc method
```
const { convertedHtml, originalHtml } = await md2html(md, meta, data, docPageType.DOC_ARTICLE);
```

Reply:
Done.